src/udiskslinuxmanager.c

```diff
@@ -765,7 +766,16 @@ handle_mdraid_create (UDisksManager         *_object,
     }
   disks[disks_top] = NULL;
 
-  if (!bd_md_create (array_name, arg_level, disks, 0, NULL, FALSE, arg_chunk, NULL, &error))
+  g_variant_lookup (arg_options, "bitmap", "^&ay", &option_bitmap);
+  if (!(g_strcmp0 (option_bitmap, "none") == 0 || g_strcmp0 (option_bitmap, "internal") == 0))
```

Comment:
One more thing, revealed by the failed tests - in case of a _NULL_ value this check should be skipped.